### PR TITLE
c396382 (firebird)

### PR DESCRIPTION
**Pull request by PalinaDzmitrovich**
https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4450/allure/

https://foliotest.testrail.io/index.php?/cases/view/396382

### CODE REVIEW

File: cypress/e2e/data-export/cql-filtering-job-profiles.cy.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+let user;
+
+const validFile = `autoTestValidFile${GetRandomPostfix()}.cql`;
+
+describe('data-export', () => {
+  before('Create test data', () => {
+    cy.createTempUser([
+      Permissions.inventoryAll.gui,
+      Permissions.dataExportAll.gui,
+      Permissions.dataExportEnableModule.gui,
```
Comment (nayimovag) [inline cypress/e2e/data-export/cql-filtering-job-profiles.cy.js:21]: Test Case only requires "UI: Data export module is enabled" permission

**Review by nayimovag — CHANGES_REQUESTED** [review]

File: cypress/e2e/data-export/cql-filtering-job-profiles.cy.js (hunk 1 above)
Comment (PalinaDzmitrovich) [reply cypress/e2e/data-export/cql-filtering-job-profiles.cy.js:21]: Thank you, removed

Comment (nayimovag) [reply cypress/e2e/data-export/cql-filtering-job-profiles.cy.js]: in the permissions.js it's called a bit different: dataExportEnableApp

Comment (nayimovag) [reply cypress/e2e/data-export/cql-filtering-job-profiles.cy.js]: replace

Comment (PalinaDzmitrovich) [reply cypress/e2e/data-export/cql-filtering-job-profiles.cy.js]: Thank you, done

File: cypress/support/fragments/data-export/selectJobProfile.js [hunk 2]
```diff
@@ -26,7 +36,48 @@ export default {
...
-  verifySearchButton() {
-    cy.expect(searchButton.exists());
+  verifySearchButton(isDisabled) {
+    cy.expect(Button('Search', { disabled: isDisabled }).exists());
+  },
+
+  searchForAJobProfile(profile) {
+    cy.get(jobProfilescSearchId).type(profile);
+    cy.get(xIconSelector).should('exist');
+    cy.do([Button('Search', { disabled: false }).click()]);
```
Comment (nayimovag) [inline cypress/support/fragments/data-export/selectJobProfile.js:46]: move all the Button('Search') to a const variable

Comment (PalinaDzmitrovich) [reply cypress/support/fragments/data-export/selectJobProfile.js]: Done.

**Review by nayimovag — APPROVED** [review]

File: cypress/support/fragments/data-export/selectJobProfile.js [hunk 3]
```diff
@@ -1,10 +1,20 @@
...
+  MultiColumnListCell,
+} from '../../../../interactors';
 
 const searchResults = MultiColumnList({ id: 'search-results-list' });
 const jobProfilesPaneHeader = PaneHeader({ id: 'paneHeaderpane-results-subtitle' });
 const searchField = '[class^=formControl]';
 const searchIcon = TextFieldIcon();
-const searchButton = Button('Search', { disabled: true });
+const jobProfilescSearchId = '#input-search-field';
+const xIconSelector = '[class^="endControls"]>div>div';
```
Comment (IhorBohdan) [inline cypress/support/fragments/data-export/selectJobProfile.js:15]: Maybe there's another way to use this selector? I'm confused about div's.

**Review by IhorBohdan — APPROVED** [review]

File: cypress/support/fragments/data-export/selectJobProfile.js (hunk 3 above)
Comment (PalinaDzmitrovich) [reply cypress/support/fragments/data-export/selectJobProfile.js:15]: I haven't found the way to do so
Open to any suggestions!

**Pull request merged**